hq/app/controllers/SnykController.scala

```diff
@@ -60,14 +65,11 @@ class SnykController(val config: Configuration, val configraun: com.gu.configrau
     case Right(a:String) => Attempt.Right(SnykToken(a))
   }
 
-  def getSnykOrganisation: Attempt[SnykOrganisationName] = configraun.getAsString("/snyk/organisation") match {
+  def getSnykOrganisation: Attempt[SnykGroupId] = configraun.getAsString("/snyk/organisation") match {
     case Left(a:ConfigraunError) =>
       val failure = Failure(a.message, "Could not read Snyk organisation from aws parameter store", 500, None, Some(a.e))
       Attempt.Left(failure)
-    case Right(a:String) => Attempt.Right(SnykOrganisationName(a))
+    case Right(a:String) => Attempt.Right(SnykGroupId(a))
```

Comment:
I doubt you need to annotate `a`, but if you do space after the `:`. Also, `a` is normally used for things of type `A`. This is a string so it probably has a better name? `id`?